backend/src/Prelude/Prelude.fs

```diff
@@ -397,3 +397,18 @@ let randomString (length : int) : string =
 
   assertEq "randomString length is correct" result.Length length
   result
+
+
+module NumericLiteralQ =
```

Comment:
I am not sure if this is the right place for these two modules, we need them in all Builtin (U)IntX files and in TestUtils. Please let me know if I should move them elsewhere